server/Html.tsx

```diff
@@ -36,6 +39,16 @@ const polyfills = [
 
 type Props = OwnProps & typeof defaultProps;
 
+const activeSlug = (initialData: InitialData, viewData?: any) => {
```

Comment:
Maybe `getActiveSlug` (or more accurately, `getActiveSlugClassName`)? I don't have an automatic preference for prefixing helper functions with "get" but I do often find myself in this kind of trouble:

```ts
const activeSlug = activeSlug(...); // oh wait
```

so I tend to start off with "get" more often these days.